sdk/textanalytics/Azure.AI.TextAnalytics/tests/TextAnalyticsClientLiveTests.cs

```diff
@@ -423,6 +423,29 @@ public async Task AnalyzeSentimentBatchWithErrorTest()
             Assert.AreEqual(exceptionMessage, ex.Message);
         }
 
+        [Test]
+        public void AnalyzeSentimentBatchWithNullIdTestAsync()
```

Comment:
Adding test missed in last PR to make sure that `MultiLanguageInput` doesn't introduce a breaking change

Reply:
Why the `Async` suffix in test names? Other tests don't seem to have it.

Reply:
thank you! it was my bad. removed Async from the test name